mandatory/2-function-creation.js

```diff
@@ -16,7 +22,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+// create if statement
+function formatPercentage(num) {
+  const percentageSign = "%";
+  const roundedToTwo = Math.round((num + Number.EPSILON) * 100) / 100;
+  if (num > 100) {
+    let num = 100;
+    const roundedToTwo = Math.round((num + Number.EPSILON) * 100) / 100;
```

Comment:
I love the way you though about your anwser and choose a pragmatic choice. I would have added comments here to make your logic clear and refer people to whereever you got the answer from but that's subjective